scripts/import/import_clinvar_xml

```diff
@@ -274,7 +286,7 @@ sub get_set{
 
   if (defined $set_id->[0]->[0] ){
     ## remove old set content
-    $set_del_sth->execute($set_id->[0]->[0]) if defined $clean;
+    $set_del_sth->execute($set_id->[0]->[0]) if defined $clean && $set eq 'ph_omim';
```

Comment:
It's confusing to only delete the OMIM set here. Is it safe to continue to delete all the ClinVar sets here? If not, a comment would be useful.

Reply:
Thanks for checking @sarahhunt. Have removed it in the next commit.